test/unittests/mongoBackend/mongoSubscribeContext_test.cpp

```diff
@@ -7402,21 +7205,21 @@ TEST(mongoSubscribeContext, matchEntN_AttrN_TN_CNbis)
     SubscribeContextRequest  req;
     SubscribeContextResponse res;
 
+    utInit();
```

Comment:
I have take the opportunity of this PR to introduce utInit/utExit in a bunch of test that didn't have it.